gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/ApiMigration.java

```diff
@@ -156,6 +160,29 @@ private Endpoint mapEndpoint(io.gravitee.definition.model.Endpoint lb) {
             .build();
     }
 
+    private Analytics mapAnalytics(Logging v2logging) {
+        var loggingEnabled = isLoggingEnabled(v2logging);
+        var v4logging = loggingEnabled ? mapLogging(v2logging) : new io.gravitee.definition.model.v4.analytics.logging.Logging();
+        return Analytics.builder().enabled(loggingEnabled).logging(v4logging).build();
```

Comment:
In V2 we can't disable analytics. However in V4 we can so when migrating we need to enable them

```suggestion
        return Analytics.builder().enabled(true).logging(v4logging).build();
```